### DISCUSSION TITLE
Slow, slow, slow in Vivaldi browser

### DISCUSSION THREAD

**Discussion by nightsmusic** [ruffle-rs/ruffle#5041] [Q&A]
I have a game that I play that requires flash. I've installed Ruffle on the Vivaldi browser as well as on Chrome and Firefox. I'm fickle and don't stay with one browser all the time ;)  My game zips right along on Firefox and Chrome, but runs at a snail's pace on the Vivaldi browser. I'm curious as to what the difference could be. I've posted at Vivaldi as well to ask them if it's something in their browser, but wondered if anyone else has seen this problem. Ruffle is up to date and I'm running Windows 10. TY

**Comment by n0samu** [comment]
In general, if Ruffle is unexpectedly slow in a particular browser, you should check whether hardware acceleration is enabled in that browser. Without hardware acceleration, Ruffle will indeed run very slowly (and with very high CPU usage).
